ispo-working-group-charter.md

```diff
@@ -0,0 +1,39 @@
+The purpose of this working group is to support InnerSource Program Offices (ISPOs) with materials needed to establish themselves, teach and scale InnerSource in their respective company.
+This material could be directed towards senior leaders, managers, or individuals in a variety of roles.
+
+# Summary
+
+We believe that most training and teaching content can be shared across companies when those institutions share the same relevant points of context.
+While not every company is exactly the same, not every company is wholly different, either.
+For each challenge in InnerSource, we believe that there are a small number of ways of common ways to address it that should work for most companies.
+Our goal is to document these sets of solutions to challenges, make it easy for ISPOs to find which apply to them, and then to produce sharable, open source implementations of these solutions.
+These implementations will likely be documents, videos, or code, each of which is generic enough to be used across companies,
+yet specific enough that people in the company can leverage them to solve their problems without needing notable amounts of company-specific work.
+
+# Examples
+
+As an example, to the challenge of `How do I sell InnerSource to Senior Leadership?`,
+this working group could ensure that there are video clips explaining various benefits of InnerSource from a Senior Leader perspective.
+There would be different clips that highlight different benefits of InnerSource,
+and the ISPO could select those videos that highlight the benefits that their Senior Leaders care about.
+
+As another example, to the challenge of `How do I show InnerSource KPIs?`,
```

Comment:
Just as a side remark wrt. KPI: Before it comes to the challenge of how I show InnerSource KPI's, IMO there is even a (pre-)challenge about how to select/define the relevant InnerSource KPI's.

Reply:
Great point!  That one may come up for our working group, too!